common/djangoapps/enrollment/tests/fake_data_api.py

```diff
@@ -60,10 +62,14 @@ def _get_fake_enrollment(student_id, course_id):
             return enrollment
 
 
-def _get_fake_course_info(course_id):
+def _get_fake_course_info(course_id, include_expired=False):
     """Get a course from the courses array."""
+    # if verified mode is expired and include expired is false
+    # then remove the verified mode from the course.
     for course in _COURSES:
         if course_id == course['course_id']:
+            if _VERIFIED_MODE_EXPIRED and not include_expired:
```

Comment:
Following my comment above, you'd check if `course_id in _VERIFIED_MODE_EXPIRED`.